### PR TITLE
Update tag parsing to remove formatting tags

### PR DESCRIPTION
**Pull request by rachkor**
### Summary
This PR removes formatting tags like `<DT>` and extends the HTML tag formatting to the `consequence` field. 

> I assign all rights, including copyright, to any future Dradis work by myself to Security Roots.

### CODE REVIEW

File: lib/qualys/element.rb [hunk 1]
```diff
@@ -99,15 +99,20 @@ def cleanup_html(source)
       result.gsub!(/<p>/i, "\n\n")
       result.gsub!(/<br>/i, "\n")
       result.gsub!(/          /, "")
-      result.gsub!(/<a href=\"(.*?)\" target=\"_blank\">(.*?)<\/a>/i) { "\"#{$2.strip}\":#{$1.strip}" }
+      result.gsub!(/<a href=\"(.*?)\"\s?target=\"_blank\">(.*?)<\/a>/i) { "\"#{$2.strip}\":#{$1.strip}" }
```
Comment (etdsoft) [inline lib/qualys/element.rb:102]: why is this change needed?

File: lib/qualys/element.rb [hunk 2]
```diff
@@ -99,15 +99,20 @@ def cleanup_html(source)
...
       result.gsub!(/<pre>(.*?)<\/pre>/im) { |m| "\n\nbc.. #{$1.strip}\n\np.  \n" }
       result.gsub!(/<b>(.*?)<\/b>/i) { "*#{$1.strip}*" }
+      result.gsub!(/<b>|<\/b>/i, "")
       result.gsub!(/<i>(.*?)<\/i>/i) { "_#{$1.strip}_" }
+
+      result.gsub!(/<dl>|<\/dl>/i, "\n")
+      result.gsub!(/<dt>(.*?)<\/dt>/i) { "* #{$1.strip}" }
+      result.gsub!(/<DD>(.*?)<\/DD>/i) { "** #{$1.strip}" }
```
Comment (etdsoft) [inline lib/qualys/element.rb]:
```suggestion
      result.gsub!(/<dd>(.*?)<\/dd>/i) { "** #{$1.strip}" }
```

The `/i` at the end makes it case insensitive already

File: lib/qualys/element.rb [hunk 3]
```diff
@@ -99,15 +99,20 @@ def cleanup_html(source)
       result.gsub!(/<p>/i, "\n\n")
       result.gsub!(/<br>/i, "\n")
       result.gsub!(/          /, "")
-      result.gsub!(/<a href=\"(.*?)\" target=\"_blank\">(.*?)<\/a>/i) { "\"#{$2.strip}\":#{$1.strip}" }
+      result.gsub!(/<a href=\"(.*?)\"\s?target=\"_blank\">(.*?)<\/a>/i) { "\"#{$2.strip}\":#{$1.strip}" }
       result.gsub!(/<pre>(.*?)<\/pre>/im) { |m| "\n\nbc.. #{$1.strip}\n\np.  \n" }
       result.gsub!(/<b>(.*?)<\/b>/i) { "*#{$1.strip}*" }
+      result.gsub!(/<b>|<\/b>/i, "")
```
Comment (etdsoft) [inline lib/qualys/element.rb:105]: why is this needed and wouldn't it be caught by the the previous line?

Comment (rachkor) [reply lib/qualys/element.rb:105]: In the sample files we got from teams, there are several instances of `<b>` tags that are orphaned (e.g. they don't match up open/closing). The previous line finds cases where there is an opening and a closing tag to apply the bolding formatting, then this line cleans up any remaining `<b>` tags that are left over and shouldn't be subbed for the `*`.

File: lib/qualys/element.rb (hunk 1 above)
Comment (rachkor) [reply lib/qualys/element.rb:102]: Several instances in the sample file don't have the space before `target` (e.g. `<a href=""target="">`) and weren't being caught and replaced. This resolves the issue completely.

Comment (etdsoft) [reply lib/qualys/element.rb:102]: but that's an error on their side, not ours!

File: lib/qualys/element.rb (hunk 3 above)
Comment (etdsoft) [reply lib/qualys/element.rb:105]: 🤬 qualys...

File: lib/qualys/element.rb (hunk 1 above)
Comment (rachkor) [reply lib/qualys/element.rb:102]: I'd rather not have our users suffer when it's a simple regex update! 😄
